src/components/carousel.js

```diff
@@ -0,0 +1,48 @@
+import { Swiper, SwiperSlide } from "swiper/react";
+import MovieCard from "./movie-card/movie-card";
+import SwiperCore, { Pagination, Navigation } from "swiper";
+SwiperCore.use([Pagination]);
+SwiperCore.use([Navigation]);
+
+export function Carousel(items) {
+  console.log(items);
+  return (
+    <>
+      <Swiper
+        navigation={true}
+        spaceBetween={30}
+        slidesPerView={5}
+        pagination={{}}
+        breakpoints={{
```

Comment:
I notice we have both a breakpoints property here and we have breakpoints defined in the css. Which part actually does all the styling? We don't want to have both for this, instead lets use just one. My personal preference would be using these properties instead of css. The reason is that we can then at one point programatically define all of these per Carousel component.